install/update_convert_cms_explorer_dicts.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env sh
 # Description: This script grabs all the excellent CMS Explorer dictionaries, updates them and converts them into DirBuster format (much faster than CMS Explorer)
 
+
+# bring in the color variables: `normal`, `info`, `warning`, `danger`, `reset`
+source "$( dirname "$BASH_SOURCE{[0]}" )/colors.sh"
```

Comment:
You have referenced `$BASH_SOURCE` everything. What if the default shell is something else? For example, I use 'zsh`, so would this work for me?